sdcm/tester.py

```diff
@@ -932,12 +988,12 @@ def get_cluster_k8s_gce_minikube(self):
         # This should remove some of the unpredictability of pods startup time.
         self.k8s_cluster.docker_pull(f"{self.params.get('docker_image')}:{self.params.get('scylla_version')}")
 
-        self.db_cluster = \
-            minikube.MinikubeScyllaPodCluster(k8s_cluster=self.k8s_cluster,
-                                              scylla_cluster_name=self.params.get("k8s_scylla_cluster_name"),
-                                              user_prefix=self.params.get("user_prefix"),
-                                              n_nodes=self.params.get("n_db_nodes"),
-                                              params=self.params)
+        self.db_cluster = minikube.RemoteMinimalScyllaPodCluster(
+            k8s_cluster=self.k8s_cluster,
+            scylla_cluster_name=self.params.get("k8s_scylla_cluster_name"),
+            user_prefix=self.params.get("user_prefix"),
+            n_nodes=self.params.get("n_db_nodes"),
+            params=self.params)
```

Comment:
Revert this one too and everything else LGTM.

Reply:
Not doable, minikube.MinikubeScyllaPodCluster is much shorter than minikube.RemoteMinimalScyllaPodCluster, as result no matter what you do line indent got changed.

Reply:
Ok, haven't noticed it, thanks.